tests/nightly/test_large_array.py

```diff
@@ -279,6 +279,33 @@ def test_diag():
     assert_almost_equal(r.asnumpy(), np.diag(a_np, k=k))
 
 
+def create_large_tensor():
```

Comment:
can we make this method more generic, so user can pass in values to create tensors with different dimension? Also, if it is limited to a 2D tensor, maybe you want to rename the method too.

Reply:
this is currently generic to 2D tensor. Can be used for other tests will update that in a separate PR. Apart from this do you think its good to go ?